apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -65,6 +65,31 @@ function deleteTestUser(user) {
 	});
 }
 
+function createChannel(userCredentials, name) {
+	return new Promise((resolve, reject) => {
+		request
+			.post(api('channels.create'))
+			.set(userCredentials)
+			.send({
+				name,
+			})
+			.then((res) => resolve(res.body.channel._id))
+			.catch(reject);
+	});
+}
```

Comment:
you can use async/await here:

```suggestion
async function createChannel(userCredentials, name) {
	const res = await request
		.post(api('channels.create'))
		.set(userCredentials)
		.send({
			name,
		});
	return res.body.channel._id;
}
```